src/plugins/intel_gna/src/transformations/insert_copy_layer.hpp

```diff
@@ -73,6 +73,12 @@ class InsertCopyBeforeConcatLayer : public ngraph::pass::MatcherPass {
     InsertCopyBeforeConcatLayer();
 };
 
+class InsertCopyBeforeBroadcastLayer : public ngraph::pass::MatcherPass {
```

Comment:
Please added detailed description of the new transformation in the header (like it done for other transformations)

Reply:
done